docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -78,17 +78,58 @@ Use the changelog.md file for a cumulative list of all policies that are added t
 Use the *policies* folder to review the JSON for each policy that is added or updated as listed in the changelog. The filename for each policy matches the policy name listed in the changelog. Within each policy file, the JSON field names are described aptly to help you easily identify the characteristic it represents. The JSON field named searchModel.query provides the RQL for the policy.
 
 
-//[#policy-updates]
-//=== Policy Updates
+[#policy-updates]
+=== Policy Updates
 
-//No Policy Updates planned for 23.11.2.
+The following policies will be updated in Prisma Cloud:
 
-//[cols="50%a,50%a"]
-//|===
-//|POLICY UPDATES
-//|DESCRIPTION
-//|===
+[cols="50%a,50%a"]
+|===
+|POLICY UPDATES
+|DESCRIPTION
+
+
+|*AWS S3 bucket publicly readable*
+//RLP-104677
+
+|*Changes to Policy RQL*: Policy RQL has been updated to check for Authenticated Users access. Policy remediation steps have been updated.
```

Comment:
@jenjoe22  Replace "Policy RQL has been updated" with "Policy RQL will be updated..."